api-definitions/db_positions/api.yaml

```diff
@@ -290,6 +297,14 @@ definitions:
         type: string
         minLength: 1
         example: psp1234567890
+      psp_fiscal_code:
+        type: string
+        minLength: 1
+        example: PSP fiscal code
```

Comment:
Is this an example or a description of `psp_fiscal_code` ? I think the second one

Reply:
Correct, we added an actual example from the DB and put the previous value as a description.